### DISCUSSION TITLE
Feature request: Please add a Pinyin language mode without accents!

### DISCUSSION THREAD

**Discussion by molohov** [monkeytypegame/monkeytype#1719] [Ideas] [answered]
On any modern Pinyin input method like on Windows or Google Pinyin, one is not required to enter the tone/accent of the word, instead they just need to type the main characters and then select the word they want. However, accented characters are required on Monkeytype's Pinyin language mode, which makes it inapplicable to real world Pinyin and requires heavy usage of dead keys. 

Would love to see a Pinyin mode without this requirement!

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
I'm pretty sure there was an idea for there not to be accents in languages a few months ago. What Miodec said (roughly) was that there isn't a point to removing the accents because that's what defines the languages/make them special and removing them would be taking away from that specialness.

**Reply by molohov** [reply]
Fair point, but I still argue that it's not representative of how modern Pinyin is inputted. The feature request is for an additional mode anyways, so users could decide for themselves.

**Reply by frogfrogfrogfrogfrog** [reply]
#1017 I suggested the idea as well but I'm pretty sure it's not gonna be implemented

**Reply by Miodec** [reply]
Added #1755
